PlaywrigthUITests/Tests/WebTableTests.cs

```diff
@@ -38,9 +39,119 @@ public async Task VerifyTableVisible2()
 
         //TODO: automate test cases
         //Check any mandatory field
+        [Test]
+        [Description("Verify Department is mandatory field")]
+        public async Task VerifyDepartmentIsMandatoryField()
+        {
+            //Arrange
+            await DemoQAWebTablesPage.GoToDemoQaWebTablesPage();
+            await DemoQAWebTablesPage.OpenAddPopup();
+            await DemoQAWebTablesPage.RegistrationPopup.WaitForAsync(new LocatorWaitForOptions { State = WaitForSelectorState.Visible });
+            //Act
+            await DemoQAWebTablesPage.FillFirstName("John");
+            await DemoQAWebTablesPage.FillLastName("Doe");
+            await DemoQAWebTablesPage.FillEmail("jon@doe.com");
+            await DemoQAWebTablesPage.FillAge("33");
+            await DemoQAWebTablesPage.FillSalary("555");
+            await DemoQAWebTablesPage.FillDepartment("");
+            await DemoQAWebTablesPage.ClickSubmit();
+            await DemoQAWebTablesPage.RegistrationPopup.WaitForAsync(new LocatorWaitForOptions { State = WaitForSelectorState.Visible });
```

Comment:
this line looks weird. I understand lines above (click Open, wait for popup)
.OpenAddPopup();
.RegistrationPopup.WaitForAsync
but this one looks wrong (click Submit which should lead to validation on already opened popup - why do we wait for opened popup to be opened?)
.ClickSubmit();
.RegistrationPopup.WaitForAsync

Reply:
redundant RegistrationPopup.WaitForAsync removed